lib/omise/resource.rb

```diff
@@ -31,23 +36,30 @@ def initialize(url, path, key)
     attr_reader :uri, :headers, :key
 
     def get(attributes = {})
-      @resource.get(params: attributes) { |r| Omise::Util.load_response(r) }
+      resource.get(params: attributes) { |r| load_response(r) }
```

Comment:
I'd prefer `@resource` to remain as an instance variable. It makes it more obvious that it's a private variable that may only be accessed internally in the `Resource` class.